hvplot/converter.py

```diff
@@ -519,8 +519,14 @@ def _process_symmetric(self, symmetric, clim, check_symmetric_max):
         else:
             return
 
-        cmin = np.nanquantile(data, 0.05)
-        cmax = np.nanquantile(data, 0.95)
+        if hasattr(data, 'isna'):
+            finite = data[~data.isna()]
+        elif self.gridded_data:
+            finite = data.flat[np.isfinite(data.flat)]
+        else:
+            finite = data[np.isfinite(data)]
+        cmin = np.quantile(finite, 0.05)
+        cmax = np.quantile(finite, 0.95)
```

Comment:
This seems like something to wrap up as a utility function.

Reply:
I've given up on this, it's too tricky as there are too many potential edge cases.